core-build/gulp-core-build-karma/gulpfile.js

```diff
@@ -1,5 +1,5 @@
 'use strict';
 
 let build = require('@microsoft/node-library-build');
-
+build.mocha.enabled = false;
 build.initialize(require('gulp'));
```

Comment:
Why do you have to disable mocha everywhere?

Reply:
Something changed when I updated mocha and basically it fails if it can't find any tests.